### PR TITLE
Updated README.md - added link to email lists and fixed typo.

### PR DESCRIPTION
**Pull request by billnitzberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ PBS Professional® software optimizes job scheduling and workload management in
 
 ## Community and Ways to Participate
 
-PBS Professional is a community effort and there are a variety of ways to engage, from helping answer questions to benchmarking to developing new capabilities and tests.  We value being aggressively open and inclusive, but also aggressively respectful and professional (see our [Code of Conduct](https://pbspro.atlassian.net/wiki/display/PBSPro/Code+of+Conduct).
+PBS Professional is a community effort and there are a variety of ways to engage, from helping answer questions to benchmarking to developing new capabilities and tests.  We value being aggressively open and inclusive, but also aggressively respectful and professional (see our [Code of Conduct](https://pbspro.atlassian.net/wiki/display/PBSPro/Code+of+Conduct)).
 
-COMING SOON:  The best place to start is by joining the mailing lists.  You can <JOIN HERE> and view the <ARCHIVES HERE>:
+The best place to start is by joining the mailing lists.  You can subscribe and view the archives via [PBS Pro Community](http://community.pbspro.org).
```
Comment (mike0042) [inline README.md]: You *may* subscribe

File: README.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ PBS Professional® software optimizes job scheduling and workload management in
 
 ## Community and Ways to Participate
 
-PBS Professional is a community effort and there are a variety of ways to engage, from helping answer questions to benchmarking to developing new capabilities and tests.  We value being aggressively open and inclusive, but also aggressively respectful and professional (see our [Code of Conduct](https://pbspro.atlassian.net/wiki/display/PBSPro/Code+of+Conduct).
+PBS Professional is a community effort and there are a variety of ways to engage, from helping answer questions to benchmarking to developing new capabilities and tests.  We value being aggressively open and inclusive, but also aggressively respectful and professional (see our [Code of Conduct](https://pbspro.atlassian.net/wiki/display/PBSPro/Code+of+Conduct)).
```
Comment (mike0042) [inline README.md]: I suggest ending the sentence after "professional" and start a new sentence without parentheses stating...
Refer to our code of conduct page http://...

**Comment by mike0042** [comment]
This commit should be signed with your GPG key.

**Comment by subhasisb** [comment]
@billnitzberg, when you are done updating the file, please do make sure you squash the commits to a single one (use git rebase -i) and then sign the resulting single commit (git commit --amend -S)
Since we have implemented signature checks in our build and verify system, we need every commit to be signed.

**Comment by billnitzberg** [comment]
@subhasisb Thanks for walking me through the process -- nice to see we are applying it to everyone (even ourselves).  I will do the needful.

**Comment by billnitzberg** [comment]
Closing -- will submit a well-formed pull request to replace this one.

**Pull request closed without merging**